src/classes/yaml-utils.ts

```diff
@@ -127,17 +177,21 @@ export default class YamlUtils {
    * @param file The JSON file that is being edited to create a .yaml file
    * @returns the edited JSON file for deployment
    */
-  private static setupDeployment = (args: any, file: any): any => {
+  private static setupDeployment = (args: KubeArgs, namespace: string, file: any): any => {
+
+    file.metadata.name = args.name;
```

Comment:
You seem to be mixing namespace and name here

Reply:
Fixed syntaxes, now pod is created directly under deployment, pod now listens to created service, created service now attempts to match pods within same namespace (project name)